components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -174,6 +174,8 @@
   private int[] tileHeight;
   private int scaleFactor;
 
+  private transient boolean isPALM = false;
```

Comment:
The transient usage here is fine as the variable is only ever used during `initFile` which means that it wont affect memo file usage.

Unless there are plans to make further use of this variable later I would be tempted to make it local to the `initFile` method would remove any confusion regarding whether or not it affects serialisation. Either approach works fine though.